src/components/countyDashboard.tsx

```diff
@@ -2,13 +2,16 @@ import { Select } from 'antd';
 import React, { useState } from 'react';
 import countyData from '../content/stateDashContent';
 import { stateData } from '../content/stateDashContent';
+import { dataTypes } from '../content/stateDashContent';
+import { dataLinks } from '../content/stateDashContent';
 const { Option } = Select;
 import styles from '../styles/dashboard.module.less';
 import IframeResizer from 'iframe-resizer-react';
 
```

Comment:
when you create a jsx/tsx component file, it is better to do capital case for first letter too. So the file name should be `CountyDashboard.tsx`